src/main/resources/view/DarkTheme.css

```diff
@@ -65,6 +65,7 @@
     -fx-text-weight: bold;
     -fx-alignment: center;
     -fx-opacity: 1;
+    -fx-wordwrap: true;
```

Comment:
I don't know how this will affect it, not too familiar with CSS.

Don't think it'll cause any major issue though.